modules/offers/client/views/offer-meet-edit.client.view.html

```diff
@@ -31,13 +31,16 @@
           <div class="panel panel-default">
             <div class="panel-heading">
               <h4 id="offerDescriptionLabel">
-                What is this about?
+                What is this about? <i>(required)</i>
               </h4>
             </div>
             <div class="panel-body">
               <div class="offer-description offer-description-"
                    aria-labelledby="offerDescriptionLabel"
-                   aria-required="false"
+                   aria-required="true"
+                   uib-tooltip="Write more please"
+                   tooltip-enable="(offerMeet.offer.description|plainTextLength)<5"
```

Comment:
I just made a variable with the description min. value. It's messed up with several controllers around meet. Could not do it the same like in the host controller. meet-add and meet-edit controllers should be joined into one controller. But I think it might be better to just rewrite it all to react eventually

Reply:
Because of the appearing unrelated issues, I'd say the extracting of the description length to one place is enough for now. Thank you for the effort in attempting this!